src/runtime_src/driver/zynq/drm/zocl/sched_exec.c

```diff
@@ -755,6 +755,60 @@ set_cmd_state(struct sched_cmd *cmd, enum cmd_state state)
 	SCHED_DEBUG("<- set_cmd_state\n");
 }
 
+/**
+ * set_cmd_ext_cu_idx() - Set external cu_idx of a command
+ *
+ * The cu_idx is reflected externally through the command packet
+ *
+ * @cmd: command object
+ * @cu_idx: CU idex
+ */
+inline void
+set_cmd_ext_cu_idx(struct sched_cmd *cmd, int cu_idx)
+{
+	int mask_idx = cu_mask_idx(cu_idx);
+	int mask_cu_idx = cu_idx_in_mask(cu_idx);
+
+	cmd->packet->data[mask_idx] &= 1 << mask_cu_idx;
+}
+
+/**
+ * set_cmd_ext_timestamp() - Set timestamp in the command packet
+ *
+ * The timestamp is reflected externally through the command packet
+ * TODO: Should have scheduler profiling solution in the future.
+ *
+ * @cmd: command object
+ * @ts: timestamp type
+ */
+inline void
+set_cmd_ext_timestamp(struct sched_cmd *cmd, enum zocl_ts_type ts)
+{
```

Comment:
Seems this function can be static

Reply:
There are a set of set_cmd_*() function should be static. How about let's clean up this file in 2019.2.

Reply:
See my latest commit.